tests/scripts/thread-cert/v1_2_LowPower_5_3_01_SSEDAttachment.py

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/env python3
+#
+#  Copyright (c) 2020, The OpenThread Authors.
+#  All rights reserved.
+#
+#  Redistribution and use in source and binary forms, with or without
+#  modification, are permitted provided that the following conditions are met:
+#  1. Redistributions of source code must retain the above copyright
+#     notice, this list of conditions and the following disclaimer.
+#  2. Redistributions in binary form must reproduce the above copyright
+#     notice, this list of conditions and the following disclaimer in the
+#     documentation and/or other materials provided with the distribution.
+#  3. Neither the name of the copyright holder nor the
+#     names of its contributors may be used to endorse or promote products
+#     derived from this software without specific prior written permission.
+#
+#  THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+#  AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+#  IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+#  ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
+#  LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+#  CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+#  SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+#  INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+#  CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+#  ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+#  POSSIBILITY OF SUCH DAMAGE.
+#
+
+import unittest
+
+from config import ADDRESS_TYPE
+from pktverify import consts
+from pktverify.packet_verifier import PacketVerifier
+
+import thread_cert
+
+LEADER = 1
+ROUTER = 2
+SSED_1 = 3
+
+# 0.5s
+CSL_PERIOD = 3125
+
+
+class LowPower_5_3_01_SSEDAttachment(thread_cert.TestCase):
+    TOPOLOGY = {
+        LEADER: {
+            'version': '1.2',
+            'name': 'LEADER',
+            'mode': 'rsdn',
+            'panid': 0xface,
+            'allowlist': [ROUTER, SSED_1]
+        },
+        ROUTER: {
+            'version': '1.2',
+            'name': 'ROUTER',
+            'mode': 'rsdn',
+            'panid': 0xface,
+            'router_selection_jitter': 1,
+            'whitelist': [LEADER],
+        },
+        SSED_1: {
+            'version': '1.2',
+            'name': 'SSED_1',
+            'mode': 's',
+            'panid': 0xface,
+            'whitelist': [LEADER],
+        },
+    }
+
+    def test(self):
+        self.nodes[LEADER].start()
+        self.simulator.go(5)
+        self.assertEqual(self.nodes[LEADER].get_state(), 'leader')
+
+        self.nodes[ROUTER].start()
+        self.simulator.go(5)
+        self.assertEqual(self.nodes[ROUTER].get_state(), 'router')
+
+        self.nodes[SSED_1].set_csl_period(CSL_PERIOD)
```

Comment:
It's difficult to understand why `3125` is 0.5s. 
What about defining `CSL_PERIOD` to 0.5 and convert `0.5` to `3125` in the implementation of `set_csl_period` ?
```suggestion
        CSL_PERIOD = 0.5
        self.nodes[SSED_1].set_csl_period(CSL_PERIOD)
```

Reply:
`CSL_PERIOD` is in units of TEN_SYMBOLS at the places, the internal state, the THCI. So I think it might be better to keep them consistent. I can change the comment to `0.5s, 3125 in units of ten symbols`.

Reply:
SGTM